arch/arm/boot/dts/overlays/i2c-rtc-common.dtsi

```diff
@@ -272,7 +288,8 @@
 		       <&pcf8523>, "reg:0",
 		       <&pcf8563>, "reg:0",
 		       <&m41t62>, "reg:0",
-		       <&rv1805>, "reg:0";
+		       <&rv1805>, "reg:0",
+			   <&s35390a>, "reg:0";
```

Comment:
The indentation seems different on this line - perhaps your editor was using 4 character TABs?